lib/matplotlib/stackplot.py

```diff
@@ -21,9 +21,9 @@ def stackplot(axes, x, *args,
 
     Parameters
     ----------
-    x : array-like (N)
+    x : (N,) array-like
 
-    y : 2D array (M, N) or sequence of array-like (N)
+    y : 2D (M, N) array or sequence of (N,) array-like
```

Comment:
```suggestion
    y : (M, N) array-like
```
- 2D not needed if we have (M, N)
- Doesn't (M, N) array-like include sequence of (N,) array-like?

Reply:
yes (twice)